v4-client-js/src/clients/modules/composer.ts

```diff
@@ -352,7 +351,7 @@ export class Composer {
         marketId,
         ticker,
         atomicResolution,
-        defaultFundingPpm,
+        defaultFundingPpm: 0, // default funding should be 0 to start.
```

Comment:
like the comment says, YuJin clarified this is supposed to be 0 for new markets starting off, unless there's a strong reason not to